cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -908,6 +913,67 @@ public KafkaBrokerConfigurationBuilder withTieredStorage(String clusterName, Tie
         return this;
     }
 
+    /**
+     * Configures the quotas based on the type of the plugin - {@link QuotasPluginKafka}, {@link QuotasPluginStrimzi}
+     *
+     * @param clusterName    Name of the cluster
+     * @param quotasPlugin   Configuration of the quotas plugin
+     * @return  Returns the builder instance
+     */
+    public KafkaBrokerConfigurationBuilder withQuotas(String clusterName, QuotasPlugin quotasPlugin) {
+        if (quotasPlugin != null) {
+            printSectionHeader("Quotas configuration");
+
+            // for the built-in Kafka quotas plugin we don't need to configure anything
+            if (quotasPlugin instanceof QuotasPluginStrimzi quotasPluginStrimzi) {
+                configureQuotasPluginStrimzi(clusterName, quotasPluginStrimzi);
+            }
+
+            writer.println();
+        }
```

Comment:
This does not make sense. Why do you print the header and footer when you don't add any values?

Reply:
Good catch, I originally thought that I will have also section for Kafka plugin specification there. Thanks :)